checkstyle/suppressions.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0"?>
+
+<!DOCTYPE suppressions PUBLIC
+        "-//Puppy Crawl//DTD Suppressions 1.1//EN"
+        "http://www.puppycrawl.com/dtds/suppressions_1_1.dtd">
+
+<suppressions>
+
+    <suppress checks="JavaNCSS"
+              files="MetricsResourceMethodApplicationListener.java"/>
```

Comment:
It looks like rather than suppressing this, we should refactor the code that's repeated 6 or 7 times into a method that we can reuse with only one or two parameters.

Reply:
Even with the refactor, I'm instantiating too many classes and getting "MetricsResourceMethodApplicationListener". This is probably fine, right? I'm not really fundamentally changing the structure of the code, just putting some of the sensors into an array.